backend/src/managedGameServers.ts

```diff
@@ -1,20 +1,38 @@
 import { IGameServer, IManagedGameServer, IManagedGameServerUpdateDto } from '../../common';
 import * as GameServer from './gameServer';
 import * as Storage from './storage';
+import { SqlAttribute, TableSchema } from './tableSchema';
 
-const FILE_NAME = 'managed_game_servers.json';
 const managedGameServers = new Map<string, IManagedGameServer>();
+const GAME_SERVERS_TABLE = 'gameServers';
 
 const write = async () => {
-	await Storage.write(FILE_NAME, Array.from(managedGameServers.values()));
+	await Storage.flushDB(GAME_SERVERS_TABLE);
+	for (const managedGameServer of managedGameServers.values()) {
+		await Storage.insertDB(
+			GAME_SERVERS_TABLE,
+			new Map<string, any>(Object.entries(managedGameServer))
+		);
+	}
 };
 
 const key = (gameServer: IManagedGameServerUpdateDto) => {
 	return gameServer.ip + ':' + gameServer.port;
 };
 
 export const setup = async () => {
-	const data = await Storage.read(FILE_NAME, [] as IManagedGameServer[]);
+	const attributes = [
+		{ name: 'ip', type: 'TEXT' },
+		{ name: 'port', type: 'INTEGER' },
+		{ name: 'rconPassword', type: 'TEXT' },
+		{ name: 'usedBy', type: 'TEXT' },
+		{ name: 'canBeUsed', type: 'INTEGER' },
+	] as SqlAttribute[];
+	const tableSchema = new TableSchema(GAME_SERVERS_TABLE, attributes, ['ip', 'port']);
+	await Storage.createTableDB(GAME_SERVERS_TABLE, tableSchema.generateCreateTableParameters());
```

Comment:
tableSchema already contains the table name, shouldn't `createTableDB` just take in a `TableSchema` arg ?